st2common/st2common/rbac/resolvers.py

```diff
@@ -769,58 +771,136 @@ def user_has_permission(self, user_db, permission_type):
 
     def user_has_resource_api_permission(self, user_db, resource_api, permission_type):
         assert permission_type in [PermissionType.API_KEY_CREATE]
-        # TODO:
-        return True
+        return self._user_has_global_permission(user_db=user_db, permission_type=permission_type)
```

Comment:
It's not being used by WebhooksController.get_all method - https://github.com/StackStorm/st2/pull/3351/commits/aef02a481fb29cfbc45e216eeac3a77782147454#diff-91ccd907e72f23cb1463520e892c92e6R98.